docs/extensions/tutorials/_posts/1970-01-01-SettingLocalEnvironment.md

```diff
@@ -21,114 +21,95 @@ Before setting up a local environment for building Shoutem extensions, set up Re
 
 In that tutorial, previewing changes in iOS apps locally is only available on Mac. That's not the case when you're using Shoutem. You can test your React Native apps on iOS platform by using the **Shoutem Preview app** (available for [iOS]({{ site.shoutem.previewAppiOS }}) and [Android]({{ site.shoutem.previewAppAndroid }})) for testing your app on a physical iOS device or just using the [Shoutem Builder]({{ site.shoutem.builderURL }}) for testing it on an iOS simulator.
 
-## Previewing apps
+## Previewing Options
```

Comment:
I think we shouldn't talk here about the `Previewing Options`. It's little bit not logic flow IMHO.

Setting local environment is for the development which also includes previewing of the app. We should rather talk about how to develop app locally. So, first explain how to develop app locally using `shoutem run` and its benefits (that you can develop on any platform - Windows, Mac and Linux and preview the changes automatically on the without pushing the app to Shoutem). Once that's explained (just rearrange the text below), explain the use case when using `shoutem run` is not enough - and that is to test native changes inside of the extension. Use the sentence you already described it in: "This is because the Shoutem Preview app has it’s own binary, so it can only preview changes made to the JavaScript bundle of the app." Say that for that reason, they could use react-native-camera, but only because it's already contained in the Shoutem Preview app, but to be sure when adding native dependencies, use `react-native` commands.

And finally, mention builder, but not within the development options, but that they can preview their extensions which don't contain native dependencies there as well.